velox/functions/remote/server/CMakeLists.txt

```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-add_library(velox_functions_remote_server RemoteFunctionService.cpp)
+velox_add_library(velox_functions_remote_server RemoteFunctionService.cpp)
 
-target_link_libraries(
+velox_link_libraries(
   velox_functions_remote_server
```

Comment:
Actually this is standalone on purpose as it's only used for development and in testing

Reply:
Oh, sorry. I didn't know it!

I've reverted this and other `remote/` change.